### PR TITLE
git-feature: add page

### PR DESCRIPTION
**Pull request by deadpyxel**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known): 6.5.0**

This is related to the open issue #5137 , documenting the git-feature command.

### CODE REVIEW

File: pages/common/git-feature.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# git pull
+
+> Create/Merge feature branch.
```
Comment (navarroaxel) [inline pages/common/git-feature.md:3]:
```suggestion
> Create or merge feature branches.
```

File: pages/common/git-feature.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+> Feature branchs obey the format feature/<name>.
+> More information: <https://github.com/tj/git-extras/blob/master/Commands.md#git-feature>.
+
+- Start a new feature branch:
```
Comment (navarroaxel) [inline pages/common/git-feature.md:7]:
```suggestion
- Create and switch to a new feature branch:
```

File: pages/common/git-feature.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+
+`git feature {{feature_name}}`
+
+- Finish the feature with --no-ff merge:
```
Comment (navarroaxel) [inline pages/common/git-feature.md:11]:
```suggestion
- Merge a feature branch into the current branch creating a merge commit:
```

File: pages/common/git-feature.md [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+
+`git feature finish {{feature_name}}`
+
+- Finish the feature with --squash merge:
```
Comment (navarroaxel) [inline pages/common/git-feature.md:15]:
```suggestion
- Merge a feature branch into the current branch squashing the changes into one commit:
```

File: pages/common/git-feature.md [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+
+`git feature finish --squash {{feature_name}}`
+
+- Publish a feature to remote repository:
```
Comment (navarroaxel) [inline pages/common/git-feature.md:19]:
```suggestion
- Send changes from a specific feature branch to its remote counterpart:
```

**Comment by deadpyxel** [comment]
Not sure what happened, I did not mean to make the previous review request to disappear

File: pages/common/git-feature.md [hunk 6]
```diff
@@ -0,0 +1,21 @@
...
+
+`git feature finish {{feature_name}}`
+
+- Merge a feature branch into the current branch squashing the changes into one commit:
+
+`git feature finish --squash {{feature_name}}`
+
+- Send changes from a specific feature branch to its remote counterpart:
+
+`git feature {{feature_name}} -r {{remote_name}}`
```
Comment (navarroaxel) [inline pages/common/git-feature.md:21]:
```suggestion
`git feature {{feature_name}} --remote {{remote_name}}`
```

**Review by kbdharun — COMMENTED** [review]
Thanks for your contribution. I have some suggestions for this page.

**Comment by deadpyxel** [comment]
Thanks for the suggestions. and sorry for the spam about the reviews, didn't know  only one reviewer was allowed

**Review by navarroaxel — APPROVED** [review]

**Comment by kbdharun** [comment]
> Thanks for the suggestions. and sorry for the spam about the reviews, I didn't know only one reviewer was allowed

I don't know why GitHub does this. When rerequesting reviews the other reviewer's request is removed. No problem, I will check this page.

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

**Pull request merged**